crates/core/src/types/account.rs

```diff
@@ -4,15 +4,35 @@ use bytes::Bytes;
 use ethereum_types::{H256, U256};
 use serde::Deserialize;
 
+use crate::rlp::encode::RLPEncode;
+
 #[allow(unused)]
 #[derive(Debug, Deserialize, PartialEq)]
 pub struct Account {
+    #[serde(flatten)]
+    pub info: AccountInfo,
     #[serde(default)]
-    pub code: Bytes,
+    pub storage: AccountStorage,
+}
+
+// We use two separate structs for better compatibility with DB structure
+#[derive(Debug, Deserialize, PartialEq)]
+pub struct AccountInfo {
     #[serde(default)]
-    pub storage: HashMap<H256, H256>,
+    pub code: Bytes,
```

Comment:
this should be the hash(code) instead of the actual code

Reply:
Fixed